test/e2e_flags.go

```diff
@@ -25,6 +25,15 @@ import (
 	"os"
 	"os/user"
 	"path"
+	"sync"
+
+	"k8s.io/klog"
+	"knative.dev/pkg/test/logging"
+)
+
+const (
+	// E2EMetricExporter is the name for the metrics exporter logger
+	E2EMetricExporter = "e2e-metrics"
```

Comment:
I presume this is used somewhere?

Reply:
roughly line 123

Reply:
Sorry I wasn't clear.
It is not used outside the package, thus the constant must be lowecased.

Reply:
Still here.

Reply:
(I had to refresh because comments weren't coming up)
I copied verbatim from serving, but `grep -r E2EMetricExporter` didn't come up with anything so I'll assume it doesn't need to be